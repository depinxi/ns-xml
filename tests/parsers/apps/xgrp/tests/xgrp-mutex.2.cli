-g mno "The following option" -h breaks "group N mutex" 