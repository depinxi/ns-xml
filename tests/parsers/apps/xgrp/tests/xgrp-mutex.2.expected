CLI: "-g", "mno", "The following option", "-h", "breaks", "group N mutex"
Value count: 1
Values: "The following option"
Error count: 1
Subcommand: 
groupJ=
switchA=False
argumentB=
groupK=
multiargC=
argumentD=
groupL=groupN
groupM=
switchE=False
argumentF=
groupN=argumentG
argumentG=mno
multiArgumentH=
groupO=argumentFdefault
argumentFdefault=Default value
argumentP=
